File: Squares_Of_A_Sorted_Array/py.d/squared_array.py
from typing import List

class Solution:
  def sortedSquares(array: List) -> List:
    return sorted([num**2 for num in array])

if __name__=="__main__":
  sol = Solution()
  print(sol.sortedSquares([-4,-1,0,3,10])) # => [0,1,9,16,100]

File: Squares_Of_A_Sorted_Array/js.d/squared_array.js
var sortedSquares = function(A) {
  A = A.map(num => num**2);
  return A.sort((a,b) => a - b);
}

console.log(sortedSquares([-4,-1,0,3,10])); // => [0,1,9,16,100]
